### PR TITLE
test(@wallet): Add test for toggling network & balance

### LINKED ISSUE #6819: Can see account balances per network

### PR DESCRIPTION
**Pull request by alaibe**
fixes #6819

### CODE REVIEW

File: test/ui-test/testSuites/suite_status/shared/steps/walletSteps.py [hunk 1]
```diff
@@ -43,6 +43,14 @@ def step(context, name, new_name):
 def step(context, name):
     _walletScreen.delete_saved_address(name)
 
+@When("the user toggle the network |any|")    
+def step(context, network_name):
+    _walletScreen.toggle_network(network_name)
+    
+@Then("the user have a positive balance of |any|")
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/shared/steps/walletSteps.py:50]:
```suggestion
@Then("the user has a positive balance of |any|")
```

File: test/ui-test/testSuites/suite_status/shared/steps/walletSteps.py [hunk 2]
```diff
@@ -43,6 +43,14 @@ def step(context, name, new_name):
 def step(context, name):
     _walletScreen.delete_saved_address(name)
 
+@When("the user toggle the network |any|")
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/shared/steps/walletSteps.py:46]:
```suggestion
@When("the user toggles the network |any|")    
```

File: ui/imports/shared/views/AssetsView.qml [hunk 3]
```diff
@@ -19,6 +19,7 @@ Item {
         id: assetListView
         anchors.fill: parent
         model: account.assets
+        objectName: "assetList"
```
Comment (jrainville) [inline ui/imports/shared/views/AssetsView.qml:22]: Maybe set a more descriptive name like `assetViewStatusListView` to avoid name clashes

**Review by jrainville — COMMENTED** [review]
Good job. Could you just fix the typos and if you want change the name

File: test/ui-test/src/screens/StatusWalletScreen.py [hunk 4]
```diff
@@ -199,6 +206,30 @@ def delete_saved_address(self, name: str):
...
+    
+    def toggle_network(self, network_name: str):
+        click_obj_by_name(MainWalletScreen.NETWORK_SELECTOR_BUTTON.value)
+
+        list = get_obj(NetworkSelectorPopup.LAYER_1_REPEATER.value)
+        for index in range(list.count):
+            if list.itemAt(index).objectName == network_name:
+                click_obj(list.itemAt(index))
+                click_obj_by_name(MainWalletScreen.ACCOUNT_NAME.value)
+                time.sleep(2)
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusWalletScreen.py:218]: Why is this needed?

**Review by noeliaSD — CHANGES_REQUESTED** [review]
Test fails for me in 2 different ways:
- It does not find the `Ropsten` network name.
- It does not find the `symbol`.

Here a video for better understanding (first 2 runs, no network found, 3rd, no symbol found):

https://user-images.githubusercontent.com/97019400/183619330-580ef6a2-74c4-4c55-8214-a4994a07dde3.mov

File: test/ui-test/src/screens/StatusWalletScreen.py (hunk 4 above)
Comment (alaibe) [reply test/ui-test/src/screens/StatusWalletScreen.py:218]: Because switching network takes time, there is async task doing all the recalculation

**Comment by alaibe** [comment]
Test might fail because of infura api key usage limit

File: test/ui-test/src/screens/StatusWalletScreen.py (hunk 4 above)
Comment (noeliaSD) [reply test/ui-test/src/screens/StatusWalletScreen.py:218]: And could we `squish.waitFor` something instead??

**Review by noeliaSD — APPROVED** [review]

**Review by Khushboo-dev-cpp — APPROVED** [review]
